### DISCUSSION TITLE
Need Help. How to make a simulation of soft tissue cutting.

### DISCUSSION THREAD

**Discussion by cauchy101** [Genesis-Embodied-AI/genesis-world#289] [General]
The behaviors of soft tissue, food, soil, sand, etc are important for robotic simulation on interaction tasks. Is it possible to build those kind of simulation durring meat cutting, soil digging, wheel-terral interacting in Genesis currently?

**Comment by zhouxian** [comment]
Yes that’s the main motivation for supporting soft body simulation

however, manipulation scene might work better, and locomotion with big scale terrain involving millions of particles could be very slow, before we merge a simulation scheme which dynamically adjust spatial resolution based on agent pose

**Reply by cauchy101** [reply]
I agree. Manipulation/soft robot itself work better. Currently I'm working on robotic meat cutting. The rigid robot manipulates a knife tool to separate meat into bones and different parts (which is needed in meat industry). The RL and simulation of meat broken behavior turn out to be hard issues. And my colleagues are studing wheel-soil/ interaction as well as plough-soil interaction durring robotic/automatic ploughing, which is a agriculture task. The simulation of breakable soft meterial is important for those kind of tasks.

**Reply by zhouxian** [reply]
This is a prior work that was using (a very very early version of ) genesis: https://roboninja.cs.columbia.edu/
might be of interest to you :)

**Reply by cauchy101** [reply]
Thanks for your information. The paper proposed a very interesting work. It is similar to meat deboning. I've tried a imitation method based on DMP and admitance control in real world scenarios. It is few-shot and the adaptation ability is limited.

**Reply by mikel-zhobro** [reply]
@cauchy101  @zhouxian 

Did you find what you were looking for? I am curious what the simulation technology progress in wheel-soil, plough-soil and cutting is.

I'm especially looking into how to simulate plants (rigid core and then the leafs and so on). If you guys have any pointer, I would really appreciate.

**Reply by cauchy101** [reply]
I leaved the lab and now focus on UGV researches. I think the paper Zhou mentioned could be a good reference for your topic. @mikel-zhobro
